sdk/tables/Azure.Data.Tables/tests/TablesRecordMatcher.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.Testing;
+
+namespace Azure.Data.Tables.Tests
+{
+    public class TablesRecordMatcher : RecordMatcher
+    {
+        public TablesRecordMatcher(RecordedTestSanitizer sanitizer) : base(sanitizer)
+        {
+            ExcludeHeaders.Add("Content-Type");
```

Comment:
Why?

Reply:
Perhaps there is a better way to deal with this, but the header differs in spacing between recorded and request:
```
System.InvalidOperationException : Unable to find a record for the request POST https://chrissprim.table.core.windows.net/Tables?$format=application%2Fjson%3Bodata%3Dfullmetadata
    Header differences:
        <Content-Type> values differ, request <application/json;odata=nometadata>, record <application/json; odata=nometadata>
```

Reply:
Hm, we might need to improve the header matching algorithm to be aware of this. File and issue please and let's reference it from this workaround.

Reply:
Created #11569